matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/call/model/MxCallImpl.kt

```diff
@@ -203,8 +203,10 @@ internal class MxCallImpl(
 
     override fun selectAnswer() {
         Timber.tag(loggerTag.value).v("select answer $callId")
-        if (isOutgoing) return
-        state = CallState.Answering
```

Comment:
do we still set this state somewhere else or does `CallState.Answering` no longer need to exist?

Reply:
It still exists and is set for incoming calls [here](https://github.com/vector-im/element-android/blob/develop/matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/call/model/MxCallImpl.kt#L179) as described in it's definition [here](https://github.com/vector-im/element-android/blob/develop/matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/call/CallState.kt#L37).

This `selectAnswer` function is actually used for outgoing calls the current state at this point is `Dialing` and stays as that until the `Connected` state.

Reply:
thanks for explaining

Reply:
I guess the initial issue was due to a quick copy paste of the `accept` fun (https://github.com/vector-im/element-android/blob/main/matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/call/model/MxCallImpl.kt#L179)